proc/proc_linux.go

```diff
@@ -309,23 +311,38 @@ func (dbp *Process) trapWait(pid int) (*Thread, error) {
 	}
 }
 
-func status(pid int) rune {
+func (dbp *Process) status(pid int) rune {
```

Comment:
If I leave wait inside Process I can remove the second argument and I don't have to add comm as a fourth one.